packages/pg-cloudflare/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "exports": {
     ".": {
-      "cloudflare": {
+      "workerd": {
         "import": "./esm/index.mjs",
         "require": "./dist/index.js"
       },
```

Comment:
I would keep both for backward compatibility

Reply:
Given that the `cloudflare` condition was only added very recently and seems broken for most people using it, I would be in favour of removing it. We don't want to encourage others to use this as an export condition for Cloudflare Workers.

Reply:
From what we can tell, this was merged in https://github.com/brianc/node-postgres/pull/3482 by @brianc, but in so doing broke the majority of Cloudflare projects given that `workerd` is the export condition expected